Sources/FluentSQL/DatabaseSchema+SQL.swift

```diff
@@ -52,8 +67,13 @@ extension DatabaseSchema.ConstraintAlgorithm {
 }
 
 extension DatabaseSchema.FieldConstraint {
+    @available(*, deprecated, renamed: "sql(unsafeRaw:)", message: "Renamed to `.sql(unsafeRaw:)`. Please use caution when embedding raw SQL.")
```

Comment:
Feel like we should have a `SQLRawSupporting` protocol for all these so we don't need to duplicate everything

Reply:
Not worth the headache at this point, though yeah that would've been good design.